src/hooks/index.jsx

```diff
@@ -16,3 +17,25 @@ export const useInstitutionIdQueryParam = () => {
 
   return addQueryParam;
 };
+
+export const useToast = () => {
+  const [showToast, setShowToast] = useState(false);
+  const [toastMessage, setToastMessage] = useState('');
+
+  const show = (message) => {
+    setToastMessage(message);
+    setShowToast(true);
+  };
+
+  const hide = () => {
+    setShowToast(false);
+    setToastMessage('');
+  };
+
+  return {
+    showToast,
+    toastMessage,
+    show,
+    hide,
+  };
```

Comment:
```suggestion
  return {
    isVisible: toast.isVisible,
    message: toast.message,
    showToast,
    hideToast
  };
```

Reply:
@01001110J comments addressed, thanks for the suggestions.